ui/app/pages/create-account/import-account/private-key.js

```diff
@@ -63,6 +64,16 @@ class PrivateKeyImportView extends Component {
     }
   }
 
+  checkInputEmpty () {
+    const input = document.getElementById('private-key-box')
```

Comment:
Generally `document.getElementById` shouldn't be used within React. React provides "refs" to give access to the underlying DOM elements, as explained here: https://reactjs.org/docs/refs-and-the-dom.html

Could you switch this to use a ref instead? And also the case in `import-account/json` as well

Reply:
Just changed it to use refs! Also changed the `createNewKeychain()` function in both files to refs instead of `document.getElementById`.

Reply:
Great, thanks!